isis/tests/CSMCameraTests.cpp

```diff
@@ -184,27 +186,72 @@ class CSMCameraFixture : public SmallCube {
       MockCsmPlugin loadablePlugin;
       loadablePlugin.registerModel(mockModelName, &mockModel);
 
+      // CSMState BLOB
+      StringBlob csmStateBlob(mockModelName, "CSMState");
+      csmStateBlob.Label() += PvlKeyword("ModelName", QString::fromStdString(mockModelName));
+      csmStateBlob.Label() += PvlKeyword("PluginName", QString::fromStdString(loadablePlugin.getPluginName()));
+      testCube->write(csmStateBlob);
+      filename = testCube->fileName();
+      testCube->close();
+      testCube->open(filename, "rw");
+    }
+};
+
+
+class CSMCameraFixture : public CSMCubeFixture {
+  protected:
+    Camera *testCam;
+
+    void SetUp() override {
+      CSMCubeFixture::SetUp();
+
       // Account for calls that happen while making a CSMCamera
       EXPECT_CALL(mockModel, getSensorIdentifier())
           .Times(2)
           .WillRepeatedly(::testing::Return("MockSensorID"));
       EXPECT_CALL(mockModel, getPlatformIdentifier())
           .Times(2)
           .WillRepeatedly(::testing::Return("MockPlatformID"));
+      EXPECT_CALL(mockModel, getReferenceDateAndTime())
+          .Times(1)
+          .WillRepeatedly(::testing::Return("20000101T115855.816"));
```

Comment:
For now, we need to use the same format as USGSCSM. Once we add the more flexible parsing this can change to the more legible version in the tests